p/i/index.php

```diff
@@ -80,7 +80,7 @@
 		// page to the users (especially non administrators).
 		echo '### Fatal error! ###<br />', "\n";
 		Minz_Log::error($error);
-		echo 'See log files for the service "journalctl -xeu freshrss" or check in /var/log if the service is not present.';
+		echo 'See log files for the service "journalctl -xeu freshrss" or check in /var/log if the service is not present';
```

Comment:
Seems the wrong way to resolve it to me (also nice timing, lol)

Reply:
![image](https://user-images.githubusercontent.com/1641362/106354228-6caecc00-62f0-11eb-8377-70d46794a6b0.png)

It looks a bit weird with the dot to me, but I do not care either way, I'll add it back to both and mention php-fpm and webserver also while I'm at it

Reply:
Added dots and made the messages better